### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by nnsW3**
This PR addresses several typographical errors across various files in the project. The changes improve readability and maintain the professional standard of the documentation and code comments.

Justification
Typographical errors, while minor, can detract from the overall quality of the project. Correcting these errors ensures clarity and professionalism, making the project more accessible and understandable for current and future contributors.

Hope it helps.

### CODE REVIEW

File: docs/docs/blog/2021-01-21-cross-chain-asset.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
+In this article, we aim to lay out a set of requirements for designing cross-chain asset exchange protocols between permissioned blockchains. First we present a canonical motivating use case and then list the various patterns that such cases will follow in real-life scenarios. We then present a short survey of existing cross-chain exchange protocols for public blockchains, following which we summarise the building blocks, core mechanisms and security properties commonly involved in the protocol designs. We then analyze the requirements imposed by permissioned blockchains in comparison to public blockchains, and discuss the additional properties with respect to these restrictions.
```
Comment (sandeepnRES) [inline docs/docs/blog/2021-01-21-cross-chain-asset.md:14]: `analyse` isn't wrong.

**Comment by sandeepnRES** [comment]
Also you need to follow the guidelines specified in `Contributing.md`, there's lots of issues please go through it.

**Review by petermetz — COMMENTED** [review]
Thank you for the contribution @nnsW3! LGTM once the comments from @sandeepnRES have been addressed!

**Comment by jagpreetsinghsasan** [comment]
> Also you need to follow the guidelines specified in `Contributing.md`, there's lots of issues please go through it.

+1
@nnsW3 please let me know if you are facing issues around this. I can help you fix these.

**Comment by nnsW3** [comment]
> > Also you need to follow the guidelines specified in `Contributing.md`, there's lots of issues please go through it.
> 
> +1 @nnsW3 please let me know if you are facing issues around this. I can help you fix these.
i would like your help, I'm currently working from browser ? could you explain me what shall I do next

**Comment by outSH** [comment]
@nnsW3 Read the `Contributing.md` file in root of the project and comply to it's requirements. In particular, you need to squash all the commits into single one, change the commit title and description and include sign-off in the commit. `Lint PR / Validate PR title (pull_request_target)` should be passing. Consult the file for more details

**Comment by RafaelAPB** [comment]
@nnsW3 thank you for your contribution! Can you please follow the suggestions of @outSH so we can merge your contribution?

**Comment by petermetz** [comment]
@nnsW3 Plus one to the above. Are you still working on this PR? If not, that's OK too I'm just asking for house-keeping purposes, we'd like to not have dangling PRs open for too long if possible. :-)

**Comment by petermetz** [comment]
Closing this due to inactivity. Please feel free to re-open if you find the time to work on it later!

**Pull request closed without merging**